storage/sealer/ffiwrapper/verifier_cgo.go

```diff
@@ -18,7 +18,15 @@ import (
 
 func (sb *Sealer) GenerateWinningPoSt(ctx context.Context, minerID abi.ActorID, sectorInfo []proof.ExtendedSectorInfo, randomness abi.PoStRandomness) ([]proof.PoStProof, error) {
 	randomness[31] &= 0x3f
-	privsectors, skipped, done, err := sb.pubExtendedSectorToPriv(ctx, minerID, sectorInfo, nil, abi.RegisteredSealProof.RegisteredWinningPoStProof) // TODO: FAULTS?
+	if len(sectorInfo) == 0 {
+		return nil, xerrors.Errorf("must provide sectors for winning post")
+	}
+	ppt, err := sectorInfo[0].SealProof.RegisteredWinningPoStProof()
+	if err != nil {
+		return nil, xerrors.Errorf("failed to convert to winning post proof: %w", err)
+	}
+
+	privsectors, skipped, done, err := sb.pubExtendedSectorToPriv(ctx, minerID, sectorInfo, nil, ppt) // TODO: FAULTS?
```

Comment:
Am I reading this correctly - there is technically no change here, right?

Reply:
Yeah, no functional change to the Winning PoSt side of things, just the refactor to actually supply the proof type to `pubExtendedSectorToPriv` (needed for the Window PoSt changes).